tools/mochaRunner/src/test.ts

```diff
@@ -15,6 +15,7 @@
  */
 
 import assert from 'assert/strict';
+
```

Comment:
The pre-commit lint hook forced me to make this (unrelated) change:

```
~/projects/puppeteer/tools/mochaRunner/src/test.ts
  17:1  error  There should be at least one empty line between import groups  import/order
```

Reply:
That's weird, I don't get this locally and CI also thinks there should be no empty line.

Reply:
I might have an outdated local setup (old deps?). Will remove it here.